generate_api_doc.py

```diff
@@ -56,7 +55,7 @@ def main():
     resources = {}
 
     del action_map['general_arguments']
-    for category, category_params in action_map.items():
+    for category, category_params in list(action_map.items()):
```

Comment:
Do we really all those `list()`? My tests suggest that no since we are in a fort loop and the iterator pattern is going to apply.